test_adjoint/test_adjoint.py

```diff
@@ -146,6 +150,7 @@ def test_adjoint_same_mesh(problem, qoi_type, debug=False):
     adj_values_expected = {}
     for field, fs in mesh_seq._fs.items():
         solve_blocks = mesh_seq.get_solve_blocks(field, 0)
+        firedrake.PETSc.Sys.Print(solve_blocks)
```

Comment:
I assume this isn't meant to be here?

Reply:
Sorry about that! Removed in a29c3e38d5b307cb5c762f84071c16ace78c5204